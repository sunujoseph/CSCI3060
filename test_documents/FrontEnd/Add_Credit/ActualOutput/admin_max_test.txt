Enter Username: 
Enter User Name To Add Credit To: 
Enter Amount To Add: 
Error: Cannot Add More Than 1000.00 Credit
Enter Amount To Add: 