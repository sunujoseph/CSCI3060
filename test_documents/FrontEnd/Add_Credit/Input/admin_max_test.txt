login
admin1
addcredit
userBS
1001
