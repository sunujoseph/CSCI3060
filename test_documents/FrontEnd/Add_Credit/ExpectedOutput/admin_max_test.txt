Enter Username: 
Enter User Name To Add Credit To: 
Enter Amount To Add: 
Error: Cannot Add More Than 1000.00 Credit
<<<<<<< HEAD
Enter Amount To Add: 
=======
Enter Amount To Add: 
>>>>>>> 9c2a426d2fb208ea0ac3da5fb156edc1d3a04f44
